test/GithubApi.Put.test.js

```diff
@@ -0,0 +1,33 @@
+const axios = require('axios');
+const { expect } = require('chai');
+
+const urlBase = 'https://api.github.com/user/following/aperdomob';
+
+const keyPath = axios.create({
+  path: urlBase,
+  headers: { Authorization: `token ${process.env.ACCESS_TOKEN}` }
+});
+
+describe('Stage 9 - PUT: Following', () => {
+  let follow;
+  before(async () => {
+    follow = await keyPath.put(urlBase);
+  });
+
+  it('2. Follow user', async () => {
+    expect(follow.status).to.equal(204);
+    expect(follow.data).to.be.eql('');
+  });
+
+  it('3. Follow check', async () => {
+    const check = await keyPath.get(urlBase);
+    expect(check.status).to.be.eql(204);
```

Comment:
and then add one additional expect to verify that in the response, is the user with login `aperdomob`